File: fapi/src/main/java/model/Bid.java
package model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.sql.Date;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Bid {

    private int id;
    private String issuer;
    private int bidNumber;
    private int amount;
    private double maxPrice;
    private double minPrice;
    private int priority;
    private Date creationDate;
    private Date dueDate;

    private String status;
    private String broker;
    private String client;
}

File: frontend/src/app/model/bid.ts
export class Bid {
  id: number;
  bidNumber: string;
  issuer: string;
  amount: number;
  minPrice: number;
  maxPrice: number;
  priority: string;
  status: string;
  creationDate: Date;
  dueDate: Date;
  broker: string;
  client: string;
}
